ui/apps/platform/src/Containers/Clusters/DelegateScanning/Components/ToggleDelegatedScanning.tsx

```diff
@@ -1,24 +1,60 @@
 import React from 'react';
-import { Card, CardBody, Checkbox } from '@patternfly/react-core';
+import { Card, CardBody, Flex, FlexItem, Radio } from '@patternfly/react-core';
 
+import FormLabelGroup from 'Components/PatternFly/FormLabelGroup';
 import { DelegatedRegistryConfigEnabledFor } from 'services/DelegatedRegistryConfigService';
 
 type ToggleDelegatedScanningProps = {
     enabledFor: DelegatedRegistryConfigEnabledFor;
-    toggleDelegation: () => void;
+    onChangeEnabledFor: (newEnabledState: DelegatedRegistryConfigEnabledFor) => void;
 };
 
-function ToggleDelegatedScanning({ enabledFor, toggleDelegation }: ToggleDelegatedScanningProps) {
+function ToggleDelegatedScanning({ enabledFor, onChangeEnabledFor }: ToggleDelegatedScanningProps) {
     return (
         <Card className="pf-u-mb-lg">
             <CardBody>
-                <Checkbox
-                    label="Enable delegated image scanning"
-                    isChecked={enabledFor !== 'NONE'}
-                    onChange={toggleDelegation}
-                    id="enabledFor"
-                    name="enabledFor"
-                />
+                <FormLabelGroup
+                    label="Delegate scanning for"
+                    fieldId="enabledFor"
+                    touched={{}}
+                    errors={{}}
+                >
+                    <Flex className="pf-u-mt-md pf-u-mb-lg">
+                        <FlexItem>
+                            <Radio
+                                label="None"
+                                isChecked={enabledFor === 'NONE'}
+                                id="choose-all-registries"
```

Comment:
Should this be something like `id="choose-none-registries"`? I'm not sure if the duplicate ID causes any issues but it will throw an a11y error.